src/providers/sh/util/error.js

```diff
@@ -64,7 +64,12 @@ async function responseError(res) {
     userError = false
   }
 
-  const err = new Error(message || 'Response error')
+  if (!message && fallbackMessage) {
+    message = fallbackMessage
+  }
+
+  const err = new Error(message ? `${message} (${res.status})` : 'Response error')
```

Comment:
When would we fall back to `Response error` ? seems like never, and it's too generic and confusing

Reply:
Fixed!